tutorials/vue/v-else-if.md

```diff
@@ -0,0 +1,57 @@
+The `v-else-if` is another directive provided by Vue.
+It must follow a `v-if` or it will not be recognized.
+`v-if` and `v-else-if` are essentially the same given the right circumstances.
```

Comment:
Lines 3-8 are very confusing and not helpful. A code example would be much better, but I'm not sure it is necessary. Just get rid of Lines 3-8.